### ISSUE TITLE
org.osgi.framework.BundleException on GlassFish server statup

### ISSUE THREAD

**Issue by jGauravGupta** [eclipse-ee4j/jaxb-ri#1454]
On the startup of Jakartafied GlassFish `6.0.0-SNAPSHOT` domain following exceptions are thrown, during the deployment of admin console app:

````
org.osgi.framework.BundleException: 
Unable to resolve org.glassfish.main.security.webservices.security [279](R 279.0): 
missing requirement [org.glassfish.main.security.webservices.security [279](R 279.0)] 
osgi.wiring.package; (&(osgi.wiring.package=com.sun.xml.ws.api.message)(version>=3.0.0)(!(version>=4.0.0))) 
	[
	caused by: Unable to resolve org.glassfish.metro.webservices-osgi [278](R 278.0): 
	missing requirement [org.glassfish.metro.webservices-osgi [278](R 278.0)] 
	osgi.wiring.package; (&(osgi.wiring.package=com.sun.istack.logging)(version>=3.0.0.M2)) 
		[
		caused by: Unable to resolve com.sun.xml.bind.jaxb-osgi [161](R 161.0): 
		missing requirement [com.sun.xml.bind.jaxb-osgi [161](R 161.0)] 
		osgi.wiring.package; (osgi.wiring.package=org.glassfish.jaxb.core.v2.runtime.output)
		]
	]
Unresolved requirements: [
[org.glassfish.main.security.webservices.security [279](R 279.0)] 
osgi.wiring.package; (&(osgi.wiring.package=com.sun.xml.ws.api.message)(version>=3.0.0)(!(version>=4.0.0)))
]
````
````
Unable to resolve org.glassfish.main.security.webservices.security [279](R 279.0): 
missing requirement [org.glassfish.main.security.webservices.security [279](R 279.0)] 
osgi.wiring.package; (&(osgi.wiring.package=com.sun.xml.ws.api.message)(version>=3.0.0)(!(version>=4.0.0))) 
	[
	caused by: Unable to resolve org.glassfish.metro.webservices-osgi [278](R 278.0): 
	missing requirement [org.glassfish.metro.webservices-osgi [278](R 278.0)] 
	osgi.wiring.package; (&(osgi.wiring.package=com.sun.xml.bind)(version>=3.0.0)(!(version>=4.0.0)))
	]
 Unresolved requirements: [[org.glassfish.main.security.webservices.security [279](R 279.0)] osgi.wiring.package; (&(osgi.wiring.package=com.sun.xml.ws.api.message)(version>=3.0.0)(!(version>=4.0.0)))]
````

Classes from package `org.glassfish.jaxb.core.v2.runtime.output` are moved to `runtime` module hence refactoring need to be done in [XMLStreamWriterOutput.java#L174](https://github.com/eclipse-ee4j/jaxb-ri/blob/3.0.0-M3-RI/jaxb-ri/runtime/impl/src/main/java/org/glassfish/jaxb/runtime/v2/runtime/output/XMLStreamWriterOutput.java#L174).

**Comment by lukasj** [comment]
well, short version is - IF jaxb gets updated in GF, Metro MUST (in 98% cases) be updated too

longer version - see https://github.com/eclipse-ee4j/metro-jax-ws/pull/137, https://github.com/eclipse-ee4j/metro-wsit/pull/82, https://github.com/eclipse-ee4j/metro-wsit/pull/84 and https://github.com/eclipse-ee4j/glassfish/pull/23034

**Comment by lukasj** [comment]
btw: that missing refactoring must be done not because of OSGi failure but because one jax-ws specific feature failing with M3 - [huge file upload](https://github.com/eclipse-ee4j/metro-jax-ws/blob/master/jaxws-ri/tests/unit/testcases/bugs/wsit1533/test-descriptor.xml) - and this was fixed in #1448

**Comment by lukasj** [comment]
closing this as this is very likely caused by an attempt to wire incompatible jaxb/jaxws versions together
